### DISCUSSION TITLE
2022年十大亮點職務看過來 AI人工智慧工程師夯

### DISCUSSION THREAD

**Discussion by kaoyihua27** [1101-GS3073-C/Python-Community-GS3073-C#197] [General]
https://money.udn.com/money/story/122329/5992939?from=edn_related_storybottom
COVID-19疫情肆虐全球至今已兩年多，而回顧2021年，國內更首度經歷了疫情帶來的三級警戒震撼，不僅改變了大眾的工作型態，更加速翻轉了各行各業的營運模式，為商機帶來更多可能性。展望後疫情時代，1111人力銀行盤點2022年「十大亮點職務」，提供正在求職路上的朋友思考抉擇。其中年底竄起的元宇宙發燒話題，讓AR/VR工程師身價水漲船高，年薪80K起跳。

**Comment by cobb110202008** [comment]
隨著時代潮流與趨勢，具有潛力的產業及工作也會跟著受益。

**Comment by wh20502** [comment]
裡面有提到房市交易熱絡，我覺得台灣不太打房的這個地方，感覺無時無刻都很熱絡，如果哪天哪間房屋公司研發了遠端看房的技術，我想未來找喜歡房子應該會非常快速，到時候應該還會再掀起一波買房潮。

**Comment by Shiba010** [comment]
AI工程師在以後看起來是不容易被AI取代的職業 ＸＤ

**Comment by ghost** [comment]
以後的熱門產業就是AI工程師了(O
